applications/GeoMechanicsApplication/custom_conditions/general_U_Pw_diff_order_condition.cpp

```diff
@@ -75,6 +75,9 @@ void GeneralUPwDiffOrderCondition::Initialize(const ProcessInfo& rCurrentProcess
         case 3: //2D L3P2
             mpPressureGeometry = GeometryType::Pointer( new Line2D2< Node<3> >(rGeom(0), rGeom(1)) );
             break;
+        case 5: //2D L5P2
```

Comment:
Right, used 2 order less, because:
1. I didn't commit 1 order less, namely Triangle2D10N, at moment, because to avoid a large commit. Will be added in the next commit.
2. As far as I understand, linear order for pressure leads to higher satiability.  So, it was intended behavior.